src/pages/History/HistoryTable/index.tsx

```diff
@@ -0,0 +1,166 @@
+/** @jsxImportSource @emotion/react */
+import React, { useMemo } from 'react';
+import { Typography } from '@mui/material';
+import { EllipseText, Table, TableProps, Token } from 'components';
+import { generateBscScanUrl, getTokenIdFromVAddress } from 'utilities';
+import { formatCoinsToReadableValue } from 'utilities/common';
+import { useTranslation } from 'translation';
+import { TokenId, Transaction } from 'types';
+import { useStyles } from './styles';
+
+export interface IHistoryTableProps {
+  transactions: Transaction[];
+}
+
+export const HistoryTableUi: React.FC<IHistoryTableProps> = ({ transactions }) => {
+  const { t } = useTranslation();
+  const styles = useStyles();
+
+  const columns = useMemo(
+    () => [
+      { key: 'id', label: t('history.columns.id'), orderable: true, align: 'left' },
+      { key: 'type', label: t('history.columns.type'), orderable: true, align: 'left' },
+      { key: 'txnHash', label: t('history.columns.txnHash'), orderable: true, align: 'left' },
+      { key: 'block', label: t('history.columns.block'), orderable: true, align: 'left' },
+      { key: 'from', label: t('history.columns.from'), orderable: true, align: 'left' },
+      { key: 'to', label: t('history.columns.to'), orderable: true, align: 'left' },
+      { key: 'amount', label: t('history.columns.amount'), orderable: true, align: 'right' },
+      { key: 'createdAt', label: t('history.columns.createdAt'), orderable: true, align: 'right' },
+    ],
+    [],
+  );
+
+  const cardColumns = useMemo(() => {
+    const newColumns = [...columns];
+    const [liquidityCol] = newColumns.splice(5, 1);
+    newColumns.splice(3, 0, liquidityCol);
+    return newColumns;
+  }, [columns]);
+
+  // Format assets to rows
+  const rows: TableProps['data'] = transactions.map(txn => [
+    {
+      key: 'id',
+      render: () => <Typography variant="small2">{txn.id}</Typography>,
+      value: txn.id,
+      align: 'left',
+    },
+    {
+      key: 'type',
+      render: () => (
+        <Token
+          variant="small2"
+          css={styles.whiteText}
+          symbol={getTokenIdFromVAddress(txn.vTokenAddress) as TokenId}
```

Comment:
The Token ID might not always correspond to its symbol, for example the casing could be different

Reply:
This  prop is mislabeled and probably didn't get updated when we established using symbol to display and id to identify. This component is manipulating the casing on this prop to use it as both

Reply:
Ok I see. I think we should update `Token`, so that it accepts a token ID and retrieves the corresponding token using `getToken` to get the symbol.
There might be cases where the symbol doesn't correspond to the ID uppercased (that's the case with the VBEP20 tokens for example: vXVS, vBNB etc.)

Reply:
Ticketed - https://app.clickup.com/t/2vdgmtp

Reply:
Thanks 👍
Just make sure to add the story points, tags and acceptance criteria. Sergio will thank you for that